helm/aws-operator/values.yaml

```diff
@@ -1,4 +1,16 @@
-namespace: giantswarm
-
-userID: 1000
-groupID: 1000
+docker:
+  image: "quay.io/giantswarm/aws-operator:[[ .DockerTag ]]"
+pod:
+  user:
+    id: 1000
+  group:
+    id: 1000
+project:
+  name: "aws-operator"
+resource:
+  default:
+    name: "aws-operator.v[[ .Version ]]"
+    namespace: "giantswarm"
+  pull-secret:
+    name: "aws-operator-pull-secret.v[[ .Version ]]"
+    namespace: "giantswarm"
```

Comment:
I tried to make those templates more generic. Maybe at some point we can generate helm templates for operator and change only `values.yaml` and some RBAC rules.